guides/plugins/plugins/checkout/cart/use-nested-line-items.md

```diff
@@ -0,0 +1,41 @@
+# Use nested line items
+
+# Overview
+
+This guide will show you how to use the nested line items in the storefront.
+
+# Prerequisites
+
+As most guides, this guide is also built upon the [Plugin base guide](../../plugin-base-guide.md), but you don't necessarily need that. It will only extend views and will show how the Custom Product plugin handle this.
```

Comment:
```suggestion
As most guides, this guide is also built upon the [Plugin base guide](../../plugin-base-guide.md), but you don't necessarily need that. It will only extend views and will show how the Custom Product plugin handles this.
```